tests/machine/test_machine_actions.py

```diff
@@ -65,6 +72,22 @@ def test_delete_one_machine(init, fetch):
     assert client.virtual_machines.delete.call_count == 1
 
 
+@patch('chaosazure.machine.actions.__fetch_machines', autospec=True)
```

Comment:
I would like a test that checks the various use cases of public/non public cloud...

so a test of the `chaosazurz.__init__.py` module itself.

Reply:
todo @xpdable

Reply:
done